hyp3_metadata/templates/readme.md.txt.j2

```diff
@@ -36,9 +36,13 @@ The source granule used to generate the products contained in this folder is:
 Please refer to the ASF Sentinel-1 RTC User Guide for in-depth guidance on the use of this dataset:
```

Comment:
Ooh, I notice that we're still pointing to the web document for the RTC product guide! I'll change that to point to the hyp3-docs version. I bet that's also in some of the xml files....